maritalk/model.py

```diff
@@ -0,0 +1,34 @@
+import requests
+
+class MariTalk:
+    def __init__(self, key: str, host: str='https://chat.maritaca.ai/api'):
+        self.key = key
+        self.host = host
+
+    def generate(self, messages: str, temperature: float=0.7, top_p: float=0.95, max_tokens: int=512, stopping_tokens: list=[]):
+        """
+        Generate a response from a list of messages.
+        """
+        body={
+            "model":"Maritalk",
+            "messages": messages,
+            "do_sample":"True",
+            "use_chat_template": "True",
```

Comment:
Aren't those supposed to be `bool`s?

Reply:
It does not matter too much since they are parsed on the API side, however i will change them to bool for clarity.